src/parser/jobs/nin/modules/TrickAttackUsage.js

```diff
@@ -32,7 +32,9 @@ export default class TrickAttackUsage extends Module {
 	_onCast(event) {
 		const action = getDataBy(ACTIONS, 'id', event.ability.guid)
 		if (event.timestamp >= this.parser.fight.start_time && action && action.onGcd && !(
-			action.id === ACTIONS.TEN.id || action.id === ACTIONS.TEN_KASSATSU.id || action.id === ACTIONS.CHI.id || action.id === ACTIONS.JIN.id)) {
+			action.id === ACTIONS.TEN.id || action.id === ACTIONS.TEN_NEW.id ||
+			action.id === ACTIONS.CHI.id || action.id === ACTIONS.CHI_NEW.id ||
```

Comment:
This is getting pretty unwieldy. Can this be converted to a MUDRAS constant array instead?

Reply:
It can, just not sure where the best place to put it would be since I wouldn't want to define the array twice. I'll think on it when it isn't 2 AM.